django/urls/converters.py

```diff
@@ -0,0 +1,48 @@
+from django.conf import settings
+from django.utils import lru_cache
+
+
+class BaseConverter(object):
+    pass
+
+
+class IntConverter(BaseConverter):
+    regex = '[0-9]+'
```

Comment:
I'm not sure. A - is much more likely to be an error than a negative number. I can't remember seeing a URL with a negative number.